release.md

```diff
@@ -0,0 +1,24 @@
+# Pylake release procedure
+
+## Preflight check
+- Update the changelog with the date.
+- Bump the version number in `__about__.py`.
+- Check whether any dependencies have changed.
```

Comment:
Add a point to review the changelog, make sure everything is clear and informative for users.

Reply:
Done.